### PR TITLE
Add extra information in CacheQuota for Alluxio to use

### PR DESCRIPTION
**Pull request by kewang1024**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-hive-common/src/main/java/com/facebook/presto/hive/CacheQuota.java
Comment (shixuan-fan) [inline presto-hive-common/src/main/java/com/facebook/presto/hive/CacheQuota.java:29]: typo: `identify` -> `identity`

**Review by shixuan-fan — COMMENTED** [review]
Curious: Why does Alluxio Cache Quota need identity (as I saw for file merge cache we only use the hash of the identity? Is it because they support multiple scope?

**Comment by kewang1024** [comment]
> Curious: Why does Alluxio Cache Quota need identity (as I saw for file merge cache we only use the hash of the identity? Is it because they support multiple scope?

Correct, because for in-house cache, we just want to quickly unblock, but for alluxio, we want to preserve the flexibility by passing all those schema/table/partition information inside

**Review by shixuan-fan — APPROVED** [review]
LGTM

**Comment by kewang1024** [comment]
@shixuan-fan Let's merge it after we get a clear signal from @apc999 to make sure it can fully unblock him

**Comment by shixuan-fan** [comment]
Sounds good. Just let me know, and @apc999 should be able to merge it by himself as well :)

**Comment by apc999** [comment]
thanks for the PR! let's hold on this patch for a few days before I merge https://github.com/Alluxio/alluxio/pull/11772 and figure out the concrete integration with Presto.

**Review by jainxrohit — APPROVED** [review]
Looks good!

**Review by apc999 — APPROVED** [review]

File: presto-cache/src/test/java/com/facebook/presto/cache/alluxio/TestAlluxioCacheConfig.java
Comment (apc999) [inline presto-cache/src/test/java/com/facebook/presto/cache/alluxio/TestAlluxioCacheConfig.java:46]: nit: keep setters alphabetical

**Pull request merged**
